regression/pages/studio/login_studio.py

```diff
@@ -0,0 +1,19 @@
+from bok_choy.promise import EmptyPromise
+from edxapp_pages.studio.login import LoginPage
+
+
+class StudioLogin(LoginPage):
```

Comment:
in here you should set the url. [Here](https://github.com/edx/edx-platform/blob/master/common/test/acceptance/pages/lms/dashboard.py#L22)'s an example from platform.